.golangci.yml

```diff
@@ -70,3 +69,12 @@ issues:
     - linters:
         - wsl
       text: "only one cuddle assignment allowed before range statement"
+
+    # These are stylecheck warnings that can be suppressed. This is temporary and will be deleted in further commits.
```

Comment:
This is a good strategy for breaking up very large changes like this, fixing issues in different commits per-issue-type.

In general, I found it useful to:

1. First commit to enable the comitter but add ignores for all failing rules
2. Loop over each rule, enabling it and fixing the issues it flags in a separate commit.